post_install.yml

```diff
@@ -4,7 +4,7 @@
   vars:
     secure: false
     ASSISTED_INSTALLER_BASE_URL: "{{ secure | ternary('https', 'http') }}://{{ hostvars['assisted_installer']['host'] }}:{{ hostvars['assisted_installer']['port'] }}/api/assisted-install/v1"
-    URL_ASSISTED_INSTALLER_CLUSTER: "{{ ASSISTED_INSTALLER_BASE_URL }}/clusters/{{ cluster_id }}"
+    URL_ASSISTED_INSTALLER_CLUSTER: "{{ ASSISTED_INSTALLER_BASE_URL }}/clusters/{{ lookup('file', '/home/ansible/crucible/fetched/cluster.txt') }}"
```

Comment:
This one was mistakenly hardcoded, however "{{ fetched_dest }}" can be used:
"{{ ASSISTED_INSTALLER_BASE_URL }}/clusters/{{ lookup('file', fetched_dest+'cluster.txt') }}"